vendor/github.com/containerd/cri-containerd/pkg/server/events.go

```diff
@@ -21,9 +21,9 @@ import (
 
 	"github.com/containerd/containerd"
 	eventtypes "github.com/containerd/containerd/api/events"
-	"github.com/containerd/containerd/api/services/events/v1"
 	containerdio "github.com/containerd/containerd/cio"
 	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/events"
```

Comment:
Do you have a `vendor.conf` change to go along with this?

Reply:
Done. :)